README.md

```diff
@@ -0,0 +1,67 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Sully's Project Skeletor :skull:
+
+> This is a "skeleton" version of the Portfolio project we're working on for the HTML & CSS module for the Microverse Curriculum.
+
+[![portfolio.png](https://i.postimg.cc/bwyVMvnY/portfolio.png)](https://postimg.cc/7fRNJwXj)
+
+# Sully's Portfolio
+
+## Built With
+
+- HTML
+- CSS
+- LOVE :heart:
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)

```

Comment:
- [ ] Please make sure to fix the live demo link or just remove its section from the README file since it's not a requirement for this project.
Kindly keep in mind that such a small mistake might leave a bad impression on future viewers.